Assets/Scripts/Models/Buildable/Components/Workshop.cs

```diff
@@ -364,7 +364,7 @@ private void HaulingJobForInputs(ProductionChain prodChain)
                                      false,
                                      false);
                         
-                        job.Description = string.Format("Hauling '{0}' to '{1}'", desiredInv, ParentFurniture.Name);
+                        job.Description = string.Format("Hauling '{0}' to '{1}'", desiredInv, ParentFurniture.GetName());
```

Comment:
it's a method because that's what it was? I have no idea, this is honestly a case of something that needs ran through the localization system, and I'm not sure if it would be best for localization to just get the Unlocalized Name and it will automatically localize it, or if it needs to be passed in already Localized (with getName getting the Localization)